libmscore/edit.cpp

```diff
@@ -2855,7 +2855,7 @@ void Score::removeChordRest(ChordRest* cr, bool clearSegment)
 
 void Score::cmdDeleteTuplet(Tuplet* tuplet, bool replaceWithRest)
       {
-      foreach(DurationElement* de, tuplet->elements()) {
+      foreach (DurationElement* de, tuplet->elements()) {
```

Comment:
I wonder whether this really is the culprit for the failing mtest. If so, more investigation is needed

Reply:
It is indeed curious. Let's wait for Travis to finish.

Reply:
Maybe a `for (DurationElement& de : tuplet->elements())` helps, if `for (DurationElement* de : tuplet->elements())` doesn't work?

Reply:
Maybe a `for (DurationElement& de : tuplet->elements())` helps, if `for (DurationElement* de : tuplet->elements())` doesn't work?

Reply:
let's try it.

Reply:
Doesn't work either. Let's just leave it as is for now.